crates/proxy-server/src/tls_helpers.rs

```diff
@@ -0,0 +1,104 @@
+//! TLS configuration helpers for the proxy server.
```

Comment:
by switching to using `kumo-server-common` as proposed above, we can then simply reference its `make_server_config` function directly rather than copying the code into this file.